src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -1420,7 +1419,8 @@ public boolean handleTokenExpiration(HttpServletRequest httpRequest, HttpServlet
     private void redirectOrRejectRequest(HttpServletRequest req, HttpServletResponse res)
             throws IOException, ServletException {
         if (req.getSession(false) != null || Strings.isNullOrEmpty(req.getHeader("Authorization"))) {
-            WebApp.get(Jenkins.get().servletContext).getSomeStapler().invoke(req, res, Jenkins.get(), getLoginUrl());
```

Comment:
@krezovic i was not familiar with this - was it essentially meant to perform a redirect?

Reply:
@mikecirioli I believe I was addressing feedback from the review where it was noted that it would be user-friendly to redirect user back to login when the token expires. Original implementation contained 401 only.